ambient.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module "isomorphic-unfetch"
+declare module "cookie"
```

Comment:
No typedefs for these, and I don't know too much about them yet, so I'm not adding them right now